source/backend/api/Models/RegionModel.cs

```diff
@@ -0,0 +1,18 @@
+namespace Pims.Api.Models
+{
+    /// <summary>
+    /// Provides a lease-oriented insurance model.
+    /// </summary>
+    public class RegionModel : BaseModel
```

Comment:
The address uses CodeTypeModel. Would that model suffice?

Reply:
yes, either codeTypeModal or just TypeModel

Reply:
Updated